test/unit/report_test.rb

```diff
@@ -113,6 +113,23 @@ def setup
       report = TestReport.new
       assert_equal({}, report.metrics)
     end
+
+    test 'can view reports as non-admin user' do
+      #Testing Issue #13920
```

Comment:
Don't reference outside sources, explain better what you want to test in the title or a comment instead